scripts/aws/ebs-warmer.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Ref: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-initialize.html
+
+function log() {
+    echo "[$(date --rfc-3339=seconds)] $1"
+}
+
+log "Warming up RWD EBS volume..."
+find /opt/rwd-data/nhd/Main_Watershed/* -type f | xargs -L1 -t cat >/dev/null
+find /opt/rwd-data/drb/Main_Watershed/* -type f | xargs -L1 -t cat >/dev/null
```

Comment:
Shellcheck complains about these lines:

```
In scripts/aws/ebs-warmer.sh line 10:
find /opt/rwd-data/nhd/Main_Watershed/* -type f | xargs -L1 -t cat >/dev/null
^-- SC2038: Use either 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow for non-alphanumeric filenames.
```

However, I wanted to echo the commands being executed as they happen (using the `xargs` `-t` flag) to give a sense of progress but I couldn't find an elegant way to do this by following the `shellcheck` suggestions.